src/scss/components/rankings.scss

```diff
@@ -3,7 +3,7 @@
 }
 
 #rankings-row:not(.logo-striped) {
-	height: calc(100vh - 6rem);
+	height: calc(100vh - 6rem - 12.5%);
```

Comment:
Why is that?

Reply:
If we set it to just 10%, the DOM goes over 100% and a scrollbar is added

Reply:
So this should be only when it's needed, not always.

Reply:
This is not always, it happens only when it's needed

Reply:
Oohhh, just got it. It's OK. my bad

Reply:
Ooohh just saw it. OK, my bad.